components/service/experiments/src/test/java/mozilla/components/service/experiments/ExperimentsTest.kt

```diff
@@ -30,25 +30,37 @@ import java.io.File
 import kotlin.reflect.full.functions
 import kotlin.reflect.jvm.isAccessible
 
+internal fun resetExperiments(
```

Comment:
nit: I think this could be `private` since it's not used outside of this file, and could be a function of the test class rather than in global scope.